cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -1034,7 +1028,7 @@ impl ManagedEnvironment {
     ///
     /// Copies the `env/` directory from the current generation to the `.flox/` directory
     /// and returns a [CoreEnvironment] for the `.flox/env`.
-    fn local_env_from_current_generation(
+    fn local_env_or_current_generation(
```

Comment:
fwiw i dont think `local_env_or_current_checkout` is really clearer since it does not imply that the local env is ensured by checking out the current generation. In fact it sounds more like the contrary, where if the local env is not present the current generation is used.

Alternative names i had in mind where `ensure_local_checkout` (simlar to ensure branch that we also have around here).
Alternatively this could even be ensured once at construction time of the managed environment and then stored as a struct member, at least that would make access in the individual methods a bit clearer.

Reply:
How about `local_env_or_copy_current_generation`?